tests/test_create_agent.py

```
@@ -0,0 +1,697 @@
+                    not agent_file.exists()
+                ), "Agent file should not be created in dry-run"
+
+
+# =============================================================================
+# Helper Functions
+# =============================================================================
+
+
+def setup_temp_project(
+    tmp_path: Path,
+    include_template: bool = True,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)